README.md

```diff
@@ -281,6 +282,22 @@ There are three different pipelines for demos use case: classification, instance
 
 ## pytorch object detection use case:
 
+There are two versions of this pipeline implemented in the code -
+
+- Using YOLOv5 from [PyTorch Hub](https://pytorch.org/hub/)
+- Using YOLOv5 from [Ultralytics](https://docs.ultralytics.com/)
+
+**Intel Optimized packages for pytorch is available only for running inference using OpenVino for Ultralytics implementation.**
+
+Performance comparison between using Intel optimzed packages for Ultralytics using OpenVino on different Intel platforms is available on the [official Ultralytics Page](https://docs.ultralytics.com/integrations/openvino/)
+
+As a developer make sure to set these two parameters in [pytorch-yolov5-env.txt](use-cases/demos/pytorch-yolov5-env.txt) -
+
+- **USE_ULTRALYTICS** - Set it to 1 to use Ultralytics pipeline else to 0
+- **INTEL_OPTIMIZED** - Set it to 1 to use Intel optimzed packages for Ultralytics implementation else to 0
+
+**Note: Default for both these are set as 0.**
```

Comment:
could you explain four cases, kind of confusing this way; are these two flags independent or they are kind of dependent?

Reply:
These two flags are dependent only for ultralytics implementation -
USE_ULTRALYTIC = 0 - use torch hub implementation (doesn't matter what INTEL_OPTIMIZED is )
USE_ULTRALYTIC = 1, INTEL_OPTIMIZED = 0 - use ultralytics implementation 
USE_ULTRALYTIC = 1, INTEL_OPTIMIZED = 1 -  use ultralytics implementation with intel optimized packages

Reply:
please put what you just explained in the docs, from the variable name itself it is not very intuitive that is the case